From 318594271b235d6e2c84ded879549a522cae6122 Mon Sep 17 00:00:00 2001
From: "Philipp A. Hartmann" <pah@qo.cx>
Date: Fri, 21 Mar 2014 17:57:16 +0100
Subject: contrib/credential/gnome-keyring/Makefile: add "install"

In order to integrate the build of the credential helper
from within a build of Git itself (e.g. for packaging),
an "install" target would be helpful.

Signed-off-by: Philipp A. Hartmann <pah@qo.cx>
---
 contrib/credential/gnome-keyring/Makefile | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/contrib/credential/gnome-keyring/Makefile b/contrib/credential/gnome-keyring/Makefile
index c3c7c98..d4b52ae 100644
--- a/contrib/credential/gnome-keyring/Makefile
+++ b/contrib/credential/gnome-keyring/Makefile
@@ -4,10 +4,14 @@ all:: $(MAIN)
 CC = gcc
 RM = rm -f
 CFLAGS = -g -O2 -Wall
+INSTALL = install
 
 -include ../../../config.mak.autogen
 -include ../../../config.mak
 
+prefix ?= /usr/local
+gitexecdir ?= $(prefix)/libexec/git-core
+
 INCS:=$(shell pkg-config --cflags gnome-keyring-1 glib-2.0)
 LIBS:=$(shell pkg-config --libs gnome-keyring-1 glib-2.0)
 
@@ -22,3 +26,7 @@ $(MAIN): $(OBJS)
 
 clean:
 	@$(RM) $(MAIN) $(OBJS)
+
+install: all
+	$(INSTALL) -d -m 0755 $(DESTDIR)$(gitexecdir)
+	$(INSTALL) -m 0755 $(MAIN) $(DESTDIR)$(gitexecdir)
-- 
1.9.1

